src/Apps/Auction/Routes/ConfirmBid/BidderPositionQuery.tsx

```diff
@@ -0,0 +1,36 @@
+import {
+  BidderPositionQuery,
+  BidderPositionQueryVariables,
+} from "__generated__/BidderPositionQuery.graphql"
+import { graphql } from "react-relay"
+import { Environment, fetchQuery } from "relay-runtime"
+
+export const bidderPositionQuery = (
+  environment: Environment,
+  variables: BidderPositionQueryVariables
+) => {
+  return fetchQuery<BidderPositionQuery>(
```

Comment:
Last time I touched this code the type signature was not right and needed to add a new type to work around. Is the issue gone now?

Reply:
I believe so. The types all looked right in VSCode and `yarn type-check` runs successfully. I saw a comment about needing a special signature until relay is upgraded, so now that we've upgraded, I think that signature is correct.